tests/test_grid.py

```diff
@@ -356,7 +356,7 @@ def test_triangulate_inplace(hexbeam):
 
 
 @pytest.mark.parametrize('binary', [True, False])
-@pytest.mark.parametrize('extension', pyvista.pointset.UnstructuredGrid._WRITERS)
+@pytest.mark.parametrize('extension', pyvista.UnstructuredGrid._WRITERS)
```

Comment:
I changed this and the other test in this module because these were the only internal uses of `pyvista.pointset`, and considering its contents it's very likely that anyone would/should be using the top-level names, as it was changed here.